src/system.h

```diff
@@ -19,18 +19,18 @@ struct Image {
     Image(wxBitmap _bm, uint64_t _hash, double _sc, vector<uint8_t> &&pd)
         : bm_orig(_bm), png_data(std::move(pd)), hash(_hash), display_scale(_sc) {}
 
-    void BitmapScale(double sc) {
+    void BitmapScale(const double &sc) {

```

Comment:
This is not an improvement.. both `double` and a `&` are 64-bit arguments and the latter is slower to access.